internal/oci/runtime_oci.go

```diff
@@ -1440,6 +1443,9 @@ func (r *runtimeOCI) defaultRuntimeArgs() []string {
 	if r.config.CgroupManager().IsSystemd() {
 		args = append(args, "--systemd-cgroup")
 	}
+	if r.config.LogLevel == "debug" {
```

Comment:
I almost wonder if this should be trace level as debug is already pretty noisy, but then again I think this would be useful. I'm not sure, any thoughts @cri-o/cri-o-maintainers ?

Reply:
> I almost wonder if this should be trace level as debug is already pretty noisy

I think so too after looking at the log of failed test logs... 
Also crun log-level is `error` by default, but there's warning logs. And runc doesn't have `log-level` option.
It would be enough to have only `error` logs maybe?